tests/rptest/services/redpanda.py

```diff
@@ -1931,6 +1964,42 @@ def check_cluster_readiness(self, cluster_name: str,
             f"Cluster {cluster_name} arrived at readyReplicas {expected_replicas}"
         )
 
+    def check_cluster_readiness_operator_v2(self, expected_replicas: int,
```

Comment:
In general, the `RedpandaServiceCloud` class should have public methods that work on all clouds & tiers. The user should not need to know if it's operator v2 etc prior to calling some methods. So it's best to expose high-level operations that make sense in both v1 and v2, and then internally it switches on v1 vs v2 if needed. Maybe this just needs to be private?